frontend/src/components/SearchTool.js

```diff
@@ -45,7 +45,6 @@ const SearchTool = (props) => {
             <Col>
               <Form.Control
                 placeholder="Min"
-                placeholder="Max"
```

Comment:
What is this removal for?

Reply:
It was causing the "Min" input field to have a placeholder of max so I removed it.  I think the placeholder="Min"(which was already there) above it is the correct one.